clevertap-core/src/main/java/com/clevertap/android/sdk/CTXtensions.kt

```diff
@@ -299,3 +305,17 @@ fun String?.toJsonOrNull(): JSONObject? {
         }
     }
 }
+
+fun View.applySystemBarsInsetsWithMargin(block : (bars:Insets, mlp:MarginLayoutParams) -> Unit) {
```

Comment:
Can we rename this `block` function parameter?

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/c98e97d0528714f07fbfeee9ea451ad99931c5c0